Client/WebFilters/ShieldStats/Httpse/HttpsEverywhereStats.swift

```diff
@@ -43,15 +45,20 @@ class HttpsEverywhereStats: LocalAdblockResourceProtocol {
         assert(httpseDb.isLoaded())
     }
     
-    func tryRedirectingUrl(_ url: URL) -> URL? {
+    func tryRedirectingUrl(_ url: URL, _ completion: @escaping(Bool) -> Void) {
         
         if url.scheme?.starts(with: "https") == true {
-            return nil
+            completion(false)
```

Comment:
Hi, you need to add a `return` statement after `completion(false)` otherwise it will fall through to the `loadDbQueue.async` code right below it.
We can also call `completion` on the same queue.

Reply:
@Brandon-T Updated as per your comments.